src/types.ts

```diff
@@ -44,6 +44,25 @@ export type ValidateReturn = { error?: ValidationError } & (
 );
 
 export interface Validator {
+  /**
+   * The type of validation: ID, tin, vat, bank,
+   */
+
+  /**
+   * The validator name, or ID's expansion name in English
+   */
+  name: string;
+
+  /**
+   * The validator name, or ID's expansion name its local name
+   */
+  localName: string;
+
+  /**
+   * The short/acronym or abbreviation of the validator
+   */
+  abbreviation?: string;
```

Comment:
What do we use to identify the type of ID when some don't have abbreviations? might make for an annoying impl. if we don't have something consistently to use reference the type, but I also see why inventing abbreviations might not be amazing given that this is a public package

Reply:
fallback to localName.